web/README.md

```diff
@@ -1,28 +1,228 @@
 # Web dashboards overview
 
-The default port is 19999; for example, to access the dashboard on localhost, use: <http://localhost:19999>
+Because Netdata is a health monitoring and _performance troubleshooting_ system,
+we put a lot of emphasis on real-time, meaningful, and context-aware charts.
 
-To view Netdata collected data you access its **[REST API v1](api/)**.
+We bundle Netdata with a dashboard and hundreds of charts, designed by both our
+team and the community, but you can also customize them yourself.
 
-For our convenience, Netdata provides 2 more layers:
+There are two primary ways to view Netdata's dashboards:
 
-1.  The `dashboard.js` javascript library that allows us to design custom dashboards using plain HTML. For information on creating custom dashboards, see **[Custom Dashboards](gui/custom/)** and **[Atlassian Confluence Dashboards](gui/confluence/)**
+1.  The [standard web dashboard](gui/) that comes pre-configured with every
+   Netdata installation. You can see it at `http://SERVER-IP:19999`, or
+   `http://localhost:19999` on `localhost`. You can customize the contents and
+   colors of the standard dashboard [using
+   JavaScript](gui/#customizing-the-standard-dashboard).
 
-2.  Ready to be used web dashboards that render all the charts a Netdata server maintains.
+2.  The [`dashboard.js` JavaScript library](#dashboardjs), which helps you
+   [customize the standard dashboards](gui/#customizing-the-standard-dashboard)
+   using JavaScript, or create entirely new [custom dashboards](gui/custom/) or
+   [Atlassian Confluence dashboards](gui/confluence/).
 
-## Customizing the standard dashboards
+You can also view all the data Netdata collects through the [REST API v1](api/).
 
-Charts information is stored at /usr/share/netdata/web/[dashboard_info.js](gui/dashboard_info.js). This file includes information that is rendered on the dashboard, controls chart colors, section and subsection heading, titles, etc.
+No matter where you use Netdata's charts, you'll want to know how to
+[use](#using-charts) them. You'll also want to understand how Netdata defines
+[charts](#charts), [dimensions](#dimensions), [families](#families), and
+[contexts](#contexts).
 
-If you change that file, your changes will be overwritten when Netdata is updated. You can preserve your settings by creating a new such file (there is /usr/share/netdata/web/[dashboard_info_custom_example.js](gui/dashboard_info_custom_example.js) you can use to start with).
+## Using charts
 
-You have to copy the example file under a new name, so that it will not be overwritten with Netdata updates.
+Netdata's chart's are far from static. They're interactive, real-time, and work
+with your mouse, touchpad, or touchscreen!
 
-To configure your info file set in `netdata.conf`:
+Hover over any chart to temporarily pause it and see the exact values presented
+as different [dimensions](#dimensions). Click or tap to lock the chart.
```

Comment:
It's unclear to someone reading this without prior experience with Netdata what `lock the chart` means.